src/interact.js

```diff
@@ -0,0 +1,22 @@
+import tasks from './tasks.js';
+import showToUi from './shouwToUi.js';
+
+const deleteAllTasks = document.querySelector('#clear-all');
+
+deleteAllTasks.addEventListener('click', () => {
+  const filterTask = tasks.filter((task) => task.completed !== true);
+  // filterTask.forEach((data, i) => {
+  //   data.index = i + 1;
+
+  // });
+  localStorage.setItem('tasks', JSON.stringify(filterTask));
+
+  window.location.reload();
+});
+
+// reloadTasks.addEventListener('click', (e) => {
+// //  showToUi()
+
+// });
+
```

Comment:
- [ ] Please remove commented codes as it makes your code messy and unreadable. :pray:

Reply:
This issue is well addressed. Nice job! :clap: